sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleController.java

```diff
@@ -129,16 +129,19 @@ private void createSample(Runnable callback, SampleDto sampleDto, Disease diseas
 	 *
 	 * @param sampleComponent
 	 *            to add the pathogen test create component to.
-	 * @param pathogenTest
-	 *            the preset values to insert. May be null.
 	 * @return the pathogen test create component added.
 	 */
-	public PathogenTestForm addPathogenTestComponent(
-		CommitDiscardWrapperComponent<? extends AbstractSampleForm> sampleComponent,
-		PathogenTestDto pathogenTest) {
+	public PathogenTestForm addPathogenTestComponent(CommitDiscardWrapperComponent<? extends AbstractSampleForm> sampleComponent) {
 
 		int caseSampleCount = caseSampleCountOf(sampleComponent.getWrappedComponent().getValue());
-		return addPathogenTestComponent(sampleComponent, pathogenTest, caseSampleCount);
+		return addPathogenTestComponent(sampleComponent, null, caseSampleCount, SormasUI::refreshView);
+	}
+
+	public PathogenTestForm addPathogenTestComponent(
```

Comment:
Please add the respective Javadoc section here, uniformly with the other definitions of `addPathogenTestComponent`.